pyscriptjs/tests/integration/test_01_basic.py

```diff
@@ -266,16 +258,18 @@ def test_py_script_src_not_found(self):
         )
         assert "Failed to load resource" in self.console.error.lines[0]
 
-        expected_msg = "(PY0404): Fetching from URL foo.py failed with error 404"
-        assert any((expected_msg in line) for line in self.console.js_error.lines)
-        assert self.assert_banner_message(expected_msg)
+        # TODO: we need to be sure errors make sense from both main and worker worlds
+        # expected_msg = "(PY0404): Fetching from URL foo.py failed with error 404"
+        # assert any((expected_msg in line) for line in self.console.js_error.lines)
+        # assert self.assert_banner_message(expected_msg)
 
-        pyscript_tag = self.page.locator("py-script")
-        assert pyscript_tag.inner_html() == ""
+        # pyscript_tag = self.page.locator("py-script")
+        # assert pyscript_tag.inner_html() == ""
 
-        self.check_js_errors(expected_msg)
+        # self.check_js_errors(expected_msg)
 
-    @pytest.mark.skip("DIFFERENT BEHAVIOUR?: we don't expose pyscript on window")
+    # TODO: ... and we shouldn't: it's a module and we better don't leak in global
```

Comment:
to clarify on this, any script can also have any config or any version because we have multiple scripts and multiple env now ... this test was based on "classic" but the global leak makes little sense in next.